test/index.test.js

```diff
@@ -205,6 +205,104 @@ describe('Dynastar - index.js', function () {
     });
   });
 
+  describe('key builders', function () {
+    let myModel, myHashKey, myRangeKey;
+
+    before(function () {
+      myHashKey = 'key';
+      myRangeKey = 'what';
+
+      myModel = dynamo.define('test2', {
+        hashKey: myHashKey,
+        rangeKey: myRangeKey,
+        schema: {
+          key: Joi.string(),
+          hello: Joi.string(),
+          what: Joi.string(),
+          other: Joi.string().allow(null)
+        }
+      });
+    });
+
+    it('supports createKey for building the hash key', function (done) {
+      const spec = { hello: 'world', what: 'thing' };
+      const myWrapped = new Dynastar({
```

Comment:
IMHO, make `myWrapped` be a closure variable (ala model/hash/rangekeys) and then in an `afterEach` drop the tables if `myWrapped` exists.  That way the tests aren't accidentally leaking into each other.